runner.go

```diff
@@ -3,31 +3,30 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"strings"
 )
 
 type runParams struct {
 	Tags          []string `json:"tags,omitempty"`
 	SmartFolderID int      `json:"smart_folder_id,omitempty"`
+	SiteID        int      `json:"site_id,omitempty"`
 }
 
 type runResponse map[string]interface{}
 
-type flagParser interface {
-	StringSlice(string) []string
-	Int(string) int
-}
-
-func createRun(f flagParser) {
-	params := makeParams(f)
+func createRun() {
+	params := makeParams()
 	resBody := postRun(params)
 
 	fmt.Println(resBody)
 }
 
-func makeParams(c flagParser) *runParams {
+func makeParams() *runParams {
+	slicedTags := strings.Split(tags, ",")
```

Comment:
Will this work if you have tags separated by a space and comma? kind, of, like, this

I think the API takes care of it so not sure if we need to worry or not... probably makes sense to take care of it in both places just in case.